src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDoclet.java

```diff
@@ -367,6 +371,18 @@ private void copyLegalFiles(boolean includeJQuery) throws DocletException {
                     }
                     DocPath filePath = DocPaths.LEGAL.resolve(entry.getFileName().toString());
                     DocFile df = DocFile.createFileForOutput(configuration, filePath);
+                    if (isDefault) {
+                        Pattern pattern = Pattern.compile("Please see (.*)\\R"); // follow jlink descriptive links
+                        Matcher matcher = pattern.matcher(Files.readString(entry));
+                        if (matcher.find()) {
+                            try {
+                                Path descriptive_link = entry.getParent().resolve(matcher.group(1));
+                                if (Files.isRegularFile(descriptive_link)) {
+                                    entry = descriptive_link;
+                                }
+                            } catch(InvalidPathException e) {}
```

Comment:
nit
```suggestion
                            } catch (InvalidPathException e) {}
```

Reply:
Thank you for your suggestion. I fixed it.